src/SwaggerProvider.Runtime/RuntimeHelpers.fs

```diff
@@ -81,6 +86,28 @@ module RuntimeHelpers =
             if not<| isNull v
             then cnt.Add(toStringContent v, k)
         cnt
+
+    let multipartFormDataContentFromObject (object:obj) =
+        let cnt = new MultipartFormDataContent()
+        let addFileStream name (stream:IO.Stream) =
+            let filename = Guid.NewGuid().ToString() // asp.net core cannot deserialize IFormFile otherwise
+            cnt.Add(new StreamContent(stream), name, filename)
```

Comment:
Another hack is that we have to provide file have for content disposition header, otherwise asp.net core backend cannot map it to `IFormFile`, but this is not reflected in OpenAPI schema :'(

```json
"requestBody": {
          "content": {
            "multipart/form-data": {
              "schema": {
                "type": "object",
                "properties": {
                  "file": {
                    "type": "string",
                    "format": "binary"
                  }
                }
              },
              "encoding": {
                "file": {
                  "style": "form"
                }
              }
            }
          }
        },
```